app/controllers/merchants/items_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Merchants::ItemsController < ApplicationController
```

Comment:
Interesting that you made a scoped controller! I'm glad you got to play around with this. It would also to be okay to put this action in the ItemsController. You could name it something different (since there is already an index), or you could combine it with the existing index and use an if statement to control which items you show.